### PR TITLE
add testing both adoptium and adoptopenjdk

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

File: .github/workflows/build-adoptopenjdk.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      with:
+        java-version: '11.0.10'
+        java-package: jdk
+        architecture: x64
+        distribution: 'adopt'
+
+    - uses: actions/checkout@v2
+
+    - name: Build app
+      run: ./mvnw --batch-mode clean install jacoco:report jacoco:report-aggregate
```
[the remark this answers is not in the document]
Comment (johnoliver) [reply .github/workflows/build-adoptopenjdk.yml:21]: it defaults to adoptopenjdk, will need to switch that at some point

**Review by karianna — APPROVED** [review]

File: .github/workflows/build-adoptium.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Build Adoptium
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
```
Comment (gdams) [inline .github/workflows/build-adoptium.yml:6]: You could do with adding names to these job elements to prevent them just appearing as build in the UI

**Review by gdams — CHANGES_REQUESTED** [review]

File: .github/workflows/build-adoptium.yml (hunk 2 above)
Comment (gdams) [inline .github/workflows/build-adoptium.yml:6]:
```suggestion
  build:
  name: Build Adoptium
```

File: .github/workflows/build-adoptopenjdk.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: Build AdoptOpenJDK
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
```
Comment (gdams) [inline .github/workflows/build-adoptopenjdk.yml:6]:
```suggestion
  build:
  name: Build AdoptOpenJDK
```

**Review by gdams — CHANGES_REQUESTED** [review]

**Review by sxa — CHANGES_REQUESTED** [review]
ECA check failing so this shouldn't be merged yet.

Also, since it hasn't been made clear in the description, this is to support using a common codebase for the Adoptium and AdoptOpenJDK APIs

**Review by gdams — APPROVED** [review]

**Pull request merged**